ui/app/AppLayouts/Profile/stores/WalletStore.qml

```diff
@@ -20,6 +20,8 @@ QtObject {
     readonly property var combinedNetworks: networksModule.combinedNetworks
     property var selectedAccount
 
+    property var networkRPCChanged: new Object() // add network id to the object if changed
```

Comment:
This yields a warning:

![image](https://github.com/status-im/status-desktop/assets/5377645/37601ecc-3290-4af2-98ef-d9be37b73521)

```suggestion
    property var networkRPCChanged: ({}) // add network id to the object if changed
```

if you really insist it should be an `Object` :)